### PR TITLE
Fix link error.

### PR DESCRIPTION
**Pull request by chenrujun**
Related PR: https://github.com/Azure/azure-sdk-for-java/pull/14350

### CODE REVIEW

File: sdk/appconfiguration/azure-spring-cloud-feature-management/README.md [hunk 1]
```diff
@@ -235,9 +235,9 @@ The following steps demonstrate an example of a progressive rollout for a new 'B
 
 #### Targeting in an Application
 
-An example web application that uses the targeting feature filter is available in the [FeatureFlagDemo](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-samples/azure-spring-cloud-feature-management-web-sample) example project.
+An example web application that uses the targeting feature filter is available in the [example project](../../spring/azure-spring-boot-samples/azure-spring-cloud-feature-management-web-sample).
```
Comment (mssfang) [inline sdk/appconfiguration/azure-spring-cloud-feature-management/README.md:238]: we need to have complete link. Having the related reference link will cause trouble in Microsoft Doc site.

**Comment by saragluna** [comment]
Seems like we shouldn't use relative links here.

https://github.com/Azure/azure-sdk/blob/master/docs/policies/README-TEMPLATE.md#link-guidelines

https://github.com/Azure/azure-sdk-for-java/pull/13989/files#diff-bbc5dba8ff02aedfeed94fd0f102c996

File: sdk/appconfiguration/azure-spring-cloud-feature-management/README.md (hunk 1 above)
Comment (chenrujun) [reply sdk/appconfiguration/azure-spring-cloud-feature-management/README.md]: Updated. Thank you for your check.

**Review by saragluna — APPROVED** [review]
LGTM.

**Pull request merged**